android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java

```diff
@@ -1835,6 +1846,11 @@ protected void onScrollChanged(int x, int y, int oldX, int oldY) {
     }
 
     protected void dispatchEvent(WebView webView, Event event) {
+      if (event.getViewTag() == RNCWebView.INVALID_VIEW_ID) {
```

Comment:
For my understanding, do you know if the view tag is always or usually the view ID? I'm wondering if react-native sets that view tag or if it's something getting set in this react-native-webview code

Reply:
It's react-native-webview specifically.

Here's an example of an event: https://github.com/discord/react-native-webview/blob/master/android/src/main/java/com/reactnativecommunity/webview/events/TopLoadingProgressEvent.kt#L10

You'll notice the viewId is passed into the constructor of the `Event`. 
It becomes the viewTag here: https://github.com/facebook/react-native/blob/main/ReactAndroid/src/main/java/com/facebook/react/uimanager/events/Event.java#L47

Reply:
Cool! Thanks for the explanation and links